src/panels/config/integrations/ha-integration-list-item.ts

```diff
@@ -47,6 +47,7 @@ export class HaIntegrationListItem extends ListItemBase {
       )}"
     >
       <img
+        alt=""
```

Comment:
There is text accompanying this image, therefore I believe the image to be decorative.